.eslintrc.js

```diff
@@ -1,14 +1,14 @@
 module.exports = {
   root: true,
   extends: [
-    '@react-native-community',
     'plugin:react/recommended',
```

Comment:
do we want `@react-native` here too? for the config.

Reply:
nvm got it via https://github.com/bluesky-social/social-app/pull/6525

Reply:
Haha beat me to it no worries :)